src/opera/commands/notify.py

```diff
@@ -0,0 +1,141 @@
+import argparse
+import typing
+from os import path
+from pathlib import Path, PurePath
+
+import shtab
+import yaml
+from yaml import YAMLError
+
+from opera.commands.info import info
+from opera.error import DataError, ParseError
+from opera.parser import tosca
+from opera.storage import Storage
+from opera.utils import prompt_yes_no_question
+
+
+def add_parser(subparsers):
+    parser = subparsers.add_parser(
+        "notify",
+        help="Notify the orchestrator about the changes (after the deployment) and run triggers from TOSCA policies"
+    )
+    parser.add_argument(
+        "--instance-path", "-p",
+        help="Storage folder location (instead of default .opera)"
+    )
+    parser.add_argument(
+        "--inputs", "-i", type=argparse.FileType("r"),
+        help="YAML or JSON file with inputs to override the inputs supplied in deploy",
+    ).complete = shtab.FILE
```

Comment:
These are used to override the inputs that were supplied with `opera deploy`. I thought that the option would come in handy when the user wants to change some input values especially when running notify operation. Does this make sense or do you think that inputs can be dropped out of here?

Reply:
I don't think changing inputs via notify makes sense in the context of a monitoring service triggering a scaling operation, because it wouldn't know what to send, and it has no business knowing inputs anyway. We don't really expect to call notify manually, right?

Having inputs here makes things much broader than they need to be IMO. We want `notify` for scaling, and the way we want to do scaling isn't via changing inputs, but by having the implementation figure things out for itself. If we change inputs here, we make "scaling" via inputs the easiest, but incorrect way.

Reply:
Well, although we are looking more from the xOpera SaaS perspective, calling `opera notify` manually is possible in real use cases. And the user could use it for anything that he wants to on already deployed nodes if the policy on that node allows it. I've prepared some examples in https://github.com/xlab-si/xopera-examples/pull/15.

But I agree that changing inputs in `notify` should not be meant for the purposes of scaling, but rather for something else. And since we don't have any real use case for that I have removed the `--inputs/-i` parameter.